### PR TITLE
feature/WAL-429/del_button

### PR DESCRIPTION
**Pull request by hrehman-sphereon**

### CODE REVIEW

**Review by nklomp — CHANGES_REQUESTED** [review]
Isn't this simply removing all delete functionality including that of VCs?

Also the ticket has a TODO, to link it to a new ticket to implement contact deletion. You are now referring to a ticket that is already done, which isn;'t the desired outcome of the ticket

**Comment by hrehman-sphereon** [comment]
@nklomp https://sphereon.atlassian.net/browse/WAL-410 May be some incorrect link was checked. This ticket seem to be in TODO. And this is the ticket mentioned in the Jira Ticket.

I fixed the delete credential as well.

File: src/components/views/SSISwipeRowViewItem/index.tsx [hunk 1]
```diff
@@ -12,15 +12,16 @@ export interface IProps {
   viewItem: JSX.Element
   listIndex: number
   onPress: () => Promise<void>
-  onDelete: () => Promise<void>
+  onDelete?: () => Promise<void>
```
Comment (BtencateSphereon) [inline src/components/views/SSISwipeRowViewItem/index.tsx:15]: add a comment here to revert this optional behavior, WAL-410 as this onDelete should be mandatory

Comment (hrehman-sphereon) [reply src/components/views/SSISwipeRowViewItem/index.tsx]: Added.

**Review by BtencateSphereon — APPROVED** [review]

**Review by nklomp — APPROVED** [review]

**Pull request merged**